pkg/jtracer/.nocover

```diff
@@ -0,0 +1 @@
+FIXME
```

Comment:
I thought an empty file is sufficient?

Reply:
it may work, but the content of these files is printed when you run `make nocover`